client/src/lib/timer/howMuchTimeLeftOnTimer.js

```diff
@@ -0,0 +1,5 @@
+export default function howMuchTimeLeftOnTimerInMs() {
+  const curTimeInMs = Date.now();
+  const timeLeftInMs = localStorage.getItem('mosaic-timer') - curTimeInMs;
```

Comment:
I highly recommend moving any interaction with localstorage out of libraries. Make it an argument, and make a non-lib function call it with the localStorage reference. 

Also, I don't think it's worth it to have all these tiny timer functions in different files.

Reply:
Also, the string 'mosaic-timer' should definitely be a constant in one place.

Reply:
Would this timer be independent of task? Like, if a user started one task for 5 minutes, then came back for another task for some reason, would it be persisted somehow?

Reply:
The answer to your last question is yes.

Reply:
In that case I may suggest making the timer task or workspace specific, to be safe. Like {mosaicTimer: {"sdfsdf": 333}}

Reply:
Oh yeah, definitely.